acsoo/templates/project/+project.name+/setup.py.bob

```diff
@@ -1,4 +1,11 @@
-from ConfigParser import ConfigParser
+{{% if odoo.series in ['11.0'] %}}
```

Comment:
if python_version == 'python3'

Reply:
AS the configparser is needed in more than two files, I will also add it in the pre_render method.